src/Compilers/Shared/CoreClrBuildClient.cs

```diff
@@ -25,6 +26,9 @@ private CoreClrBuildClient(RequestLanguage language, CompileFunc compileFunc)
 
         internal static int Run(IEnumerable<string> arguments, RequestLanguage language, CompileFunc compileFunc)
         {
+            // Register encodings for console
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
```

Comment:
Do we still need this code then?  

https://github.com/dotnet/roslyn/blob/acc053dbd9d3cb25f086f86f36f80896c8379466/src/Compilers/Core/Portable/EncodedStringText.cs#L29